core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -100,6 +102,16 @@ class MultiCloudWorkspaceService(userInfo: UserInfo,
       _ <- traceWithParent("enableLeoInWSM", parentSpan)(_ =>
         Future(workspaceManagerDAO.enableApplication(workspaceId, wsmConfig.leonardoWsmApplicationId, userInfo.accessToken))
       )
+      _ = logger.info(s"Creating Azure relay in WSM [workspaceId = ${workspaceId}]")
+      azureRelayCreateResult <- traceWithParent("createAzureRelayInWSM", parentSpan)(_ =>
+        Future(workspaceManagerDAO.createAzureRelay(workspaceId, workspaceRequest.region, userInfo.accessToken))
+      )
+      relayJobControlId = azureRelayCreateResult.getJobReport.getId
+      _ = logger.info(s"Polling on Azure relay in WSM [workspaceId = ${workspaceId}, jobControlId = ${relayJobControlId}]")
+      _ <- traceWithParent("pollGetAzureRelayCreationStatusInWSM", parentSpan)(_ =>
+        pollWMCreation(workspaceId, relayJobControlId, userInfo.accessToken, 5 seconds,
```

Comment:
Polling here are 5 second interval instead of 2 because we expect this to take about 45 seconds.

Reply:
Agreed, we need to think about making the endpoint async. But I think that is out of the scope of this ticket. I will create a new one for it.